docs/develop/maintenance/node.md

```diff
@@ -1,3 +1,375 @@
-# Node / TypeScript Dependency Maintenance
+# Node.js / TypeScript Dependency Maintenance
 
-(Placeholder – to be populated later.)
+This guide describes the process for updating and validating Node.js and TypeScript dependencies in the monorepo. It focuses on using pnpm workspaces with a centralized `package.json` and ensuring consistent, reproducible builds across Angular, React, and Node.js applications.
+
+## Overview
+
+Node.js dependencies are managed primarily via the root `package.json` file in conjunction with pnpm workspaces. Individual projects reference shared dependencies from the root, ensuring:
+
+- Version alignment across all applications and libraries
+- Simplified upgrades and security remediation
+- Reduced dependency drift and bundle duplication
+- Efficient disk usage through symlinked node_modules
+
+### Key Files
```

Comment:
We should add a table entry for `libs/**/package.json` and `apps/**/package,json`. These local `packages.json` are used in a minority of projects, usually when a library is publishable (https://nx.dev/concepts/buildable-and-publishable-libraries) or the `package.json` is an artifacts that can be deployed, for instance as part of an AWS Lambda function. If a project has a local `package.json` but it still uses the shared (root) `node_modules`, then the project must be excluded from the pnpm workspace in `pnpm-workspace.yaml`.

```console
$ find {libs,apps}/ -name package.json
libs/amp-als/api-client-angular/package.json
libs/sage-monorepo/nx-plugin/package.json
apps/sandbox/lambda-nodejs/package.json
```